tests/test_sampling.py

```diff
@@ -187,7 +187,7 @@ class MonteCarloStandardErrorTest(chex.TestCase):
 
     def setUp(self):
         super().setUp()
-        self.key = jax.random.PRNGKey(2351235)
+        self.key = jax.random.PRNGKey(20220203)
```

Comment:
:grimacing: this does not look good. Test passing depends on the seed?

Reply:
the NUTS one is slightly flaky 😔
I think I choose this seed before for the test to pass so the problem is not limit to this PR. We should randomized the seed in the test (so each time it ran it use different seed but in a reproducible way), and make this test more robust.

Reply:
Is it the "experimental proof" or the MCSE?